Server/src/openbmp.cpp

```diff
@@ -365,9 +367,13 @@ bool ReadCmdArgs(int argc, char **argv, Config &cfg) {
  * \param [in] cfg                   Reference to configuration
  * \param [in] code                  reason code for the update
  */
+#ifndef REDIS_ENABLED
 void collector_update_msg(msgBus_kafka *kafka, Config &cfg,
                           MsgBusInterface::collector_action_code code) {
-
+#else
+void collector_update_msg(Config &cfg,
```

Comment:
>collector_update_msg [](http://example.com/codeflow?start=5&length=20)

They are function overload, so you can keep both.

Reply:
changed into override way.